### DISCUSSION TITLE
Offline use is more painful that is needs to be

### DISCUSSION THREAD

**Discussion by Doublonmousse** [penrose/penrose#1615] [General]
In the general `README.md` there is no mention of a way to edit the files offline

- This is possible when going through the `CONTRIBUTING.md` file (and the installation) THEN the editor `README.md` and then the roger `README.md` to see that it is indeed possible to compile files locally from the terminal without having to start the web based ide.

I feel like this should be put in the main `README.md`, I was put off looking at this project because on a first look it looks like this project only worked online with the penrose IDE. 

As all the tools are there for an offline experience, and I feel like the vscode extension should be able to call `roger` and compile and preview the opened project directly in the vscode editor.

**Comment by Doublonmousse** [comment]
Binary release to install the `roger` CLI locally would also go a long way to reduce friction for new users

**Comment by samestep** [comment]
This sounds useful to me! I think we'd be very open to a pull request adding this functionality; @wodeni thoughts?

**Reply by wodeni** [reply]
Yes,  thanks for the suggestion @Doublonmousse! We haven't really polished the local setup. Happy to review/help with a PR to improve the situation :D.

**Reply by wodeni** [reply]
Specifically I think this can be the next item:

> As all the tools are there for an offline experience, and I feel like the vscode extension should be able to call roger and compile and preview the opened project directly in the vscode editor.
